lib/traits/RenderableView.js

```diff
@@ -91,26 +91,46 @@ var RenderableView = _.extend({},
         },
 
         renderUnrenderedChildViews: function() {
-            _.each(this.unrenderedChildViews, function(childView) {
-                childView.renderChildViewIntoParent();
-            });
+            if (!this.unrenderedChildViews || !this.unrenderedChildViews.length) {
```

Comment:
it's possible that this.unrenderedChildViews can be undefined or []. this line checks for those 2 cases